### ISSUE TITLE
Changelog of the Release 0.9.0 shows messing the info over the PR 1512

### ISSUE THREAD

**Issue by camilamacedo86** [operator-framework/operator-sdk#1682]
The changelog of the release [v0.9.0](https://github.com/operator-framework/operator-sdk/releases/tag/v0.9.0) shows missing the info regards the PR https://github.com/operator-framework/operator-sdk/pull/1512 as its break changes as follows. 

- The package "sigs.k8s.io/controller-runtime/pkg/runtime/scheme" is deprecated. Replace this import with "sigs.k8s.io/controller-runtime/pkg/scheme" in pkg/apis/{{projectName}}/v1alpha1/register.go.
- The package "sigs.k8s.io/controller-runtime/pkg/runtime/log" is deprecated. Replace this import with "sigs.k8s.io/controller-runtime/pkg/log" in pkg/apis/{{projectName}}/v1alpha1/register.go.
- The package "sigs.k8s.io/controller-runtime/pkg/runtime/signals" is deprecated. Replace this import with "sigs.k8s.io/controller-runtime/pkg/manager/signals" in pkg/apis/{{projectName}}/v1alpha1/register.go.
- The client.List List(ctx context.Context, list runtime.Object, opts ...ListOptionFunc) error was replaced for List(ctx context.Context, opts ...ListOption, list runtime.Object) error. Following an example fix it.

Also, IMHO we should update the [doc](https://github.com/operator-framework/operator-sdk/blob/34edd0a60c876f940401020307a14e0e51de6ebb/doc/operator-scope.md#operator-scope) with the info that now we are allowed to pass a list of Namespaces to the manager as well as the following example.

```go
 // Create a new Cmd to provide shared dependencies and start components
	mgr, err := manager.New(cfg, manager.Options{
		NewCache: cache.MultiNamespacedCacheBuilder(namespaces), // Here
	})
```

**Comment by LiliC** [comment]
@camilamacedo86 that PR was not merged into master and is not in the v0.9.0 release, but in the [operator-framework:refactor/controller-runtime-v0.2.0](https://github.com/operator-framework/operator-sdk/tree/refactor/controller-runtime-v0.2.0) branch. If you want to use those changes you have to pin against that branch. We are still waiting for a stable v0.2.0 release so that is why that is not merged into master.

Closing this, as it's not an issue and nothing is missing from the CHANELOG.

**Comment by camilamacedo86** [comment]
|o| I see. It still in beta :-( https://github.com/kubernetes-sigs/controller-runtime/releases
Tks.
